docs/code-quality/ca1716.md

```diff
@@ -52,10 +52,20 @@ You can suppress a warning from this rule if you're convinced that the identifie
 
 ## Configurability
 
+### Analyzed types and members surface
```

Comment:
Let's be consistent with the heading: https://docs.microsoft.com/en-us/visualstudio/code-quality/ca1802?view=vs-2019#api-surface

```suggestion
### Api Surface
```